sklearn/preprocessing/data.py

```diff
@@ -2579,6 +2586,243 @@ def quantile_transform(X, axis=0, n_quantiles=1000,
                          " axis={}".format(axis))
 
 
+class PowerTransformer(BaseEstimator, TransformerMixin):
+    """Apply a power transform featurewise to make data more Gaussian-like.
+
+    Power transforms are a family of parametric, monotonic transformations
+    that are applied to make data more Gaussian-like. This is useful for
+    modeling issues related to heteroscedasticity (non-constant variance),
+    or other situations where normality is desired. Note that power
+    transforms do not result in standard normal distributions.
```

Comment:
(i.e. mean might be far from zero and standard deviation not one?)

Reply:
Exactly!

Reply:
I meant maybe say that explicitly ;)